specification/resourcemover/resource-manager/Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json

```diff
@@ -0,0 +1,2732 @@
+          "items": {
+            "$ref": "#/definitions/MoveResourceDependencyOverride"
+          }
+        },
+        "isResolveRequired": {
+          "description": "Gets a value indicating whether the resolve action is required over the move collection.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "errors": {
+          "description": "Defines the move resource errors.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/MoveResourceError"
+            }
+          ],
+          "readOnly": true
+        }
+      }
+    },
+    "MoveResource": {
+      "description": "Defines the move resource.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Fully qualified resource Id for the resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the resource"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the resource."
+        },
+        "properties": {
+          "$ref": "#/definitions/MoveResourceProperties"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true,
+          "description": "Metadata pertaining to creation and last modification of the resource."
+        }
+      }
+    },
+    "Summary": {
+      "description": "Summary item.",
+      "type": "object",
+      "properties": {
+        "count": {
+          "format": "int32",
+          "description": "Gets the count.",
+          "type": "integer"
+        },
+        "item": {
+          "description": "Gets the item.",
+          "type": "string"
+        }
+      }
+    },
+    "SummaryCollection": {
+      "description": "Summary Collection.",
+      "type": "object",
+      "properties": {
+        "fieldName": {
+          "description": "Gets or sets the field name on which summary is done.",
+          "type": "string"
+        },
+        "summary": {
+          "description": "Gets or sets the list of summary items.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Summary"
+          },
+          "x-ms-identifiers": [
+            "item"
+          ]
+        }
+      }
+    },
+    "MoveCollectionProperties": {
+      "description": "Defines the move collection properties.",
+      "type": "object",
+      "properties": {
+        "sourceRegion": {
+          "description": "Gets or sets the source region.",
+          "type": "string"
+        },
+        "targetRegion": {
+          "description": "Gets or sets the target region.",
+          "type": "string"
+        },
+        "moveRegion": {
+          "description": "Gets or sets the move region.",
```

Comment:
>move [](http://example.com/codeflow?start=43&length=4)

what is 'the move region' - is it the same as 'the region the resources are moving to?'

Reply:
updated the description after discussing with team. for now, this is specifically for regional to zonal VM move scenario - however, in future we will leverage this parameter for other scenarios like avset to vmss flex move etc. it indicates the region where the VM Regional to Zonal move will be conducted.

Reply:
That's a little clearer, but I'm still not sure how/why it differs from source or target region. Maybe you can make it even clearer before requesting merge.

Reply:
Or even have multiple examples which cover the different scenarios this is used for?